styleguide.config.js

```diff
@@ -141,6 +141,18 @@ module.exports = {
                 },
             ],
         },
+        {
+            name: "Popover",
+            content: "packages/wonder-blocks-popover/docs.md",
+            sections: [
+                {
+                    name: "Building Blocks",
```

Comment:
This section should have `private: true`, I believe.

Reply:
These components (PopoverContentCore and PopoverContent) are going to be exported as part of the public API. For now I'm going to leave it this way. But this might change in one of the upcoming PRs. Thanks for bringing this up!